docs/source/getting-started.md

```diff
@@ -18,7 +18,49 @@ $ conda install "menuinst>=2"
 
 ## Usage
 
-Currently, there is no CLI. This is being worked on, but for now you can use the Python API:
+### The `menuinst` CLI
+
+`menuinst` provides a CLI that can be used to install shortcuts:
+
+```shell
+usage: menuinst [-h] --prefix PREFIX (--install [PKG ...] | --remove [PKG ...]) [--root-prefix ROOT_PREFIX]
+
+options:
+  -h, --help            show this help message and exit
+  --prefix PREFIX       The prefix containing the shortcuts metadate inside `Menu`
+  --install [PKG ...]   create menu items for the given packages; if none are given, create menu items for all
+                        packages in the prefix
+  --remove [PKG ...]    remove menu items for the given packages; if none are given, remove menu items for all
+                        packages in the prefix
+  --root-prefix ROOT_PREFIX
+                        The menuinst base/root prefix
+```
+
+The CLI will look for [metadata files](./defining-shortcuts) inside the directory `${PREFIX}/Menu`.
+`PKG` is a set of metadata file names.
+
+For example, if `${PREFIX}/Menu` contains the file `package_menu.json`, its shortcuts can be
+installed with:
+
+```shell
+menuinst --prefix ${PREFIX} --install package_menu.json
+```
+
+The CLI accepts some shorthands and does not require the `.json` ending or the `_menu` suffix (the
+suffixes `-menu` and `.menu` are also ignored).
+So, the following commands are equivalent for `package_menu.json`:
+
+```shell
+menuinst --prefix ${PREFIX} --install package_menu.json
+menuinst --prefix ${PREFIX} --install package_menu
+menuinst --prefix ${PREFIX} --install package
```

Comment:
I'm not too convinced about these shorthands. Ignoring the extension only is weird, and if we only use the package name it looks like it will also install the full package.

Not super against, but I'd like these to be considered. I'm willing to compromise on accepting a direct path to a JSON, or a package name to be found in `$PREFIX/Menu/*.json`.